test/units/testsuite-60.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+set -eux
+set -o pipefail
+
+systemd-analyze log-level debug
+systemd-analyze log-target journal
+
+NUM_DIRS=20
+
+# mount/unmount enough times to trigger the /proc/self/mountinfo parsing rate limiting
+
+for ((i = 0; i < NUM_DIRS; i++)); do
+    mkdir "/tmp/meow$i"
+done
+
+for ((i = 0; i < NUM_DIRS; i++)); do
+    mount -t tmpfs tmpfs "/tmp/meow$i"
+done
+
+systemctl daemon-reload
+systemctl list-units -t mount tmp-meow* | grep tmp-meow
+
+for ((i = 0; i < NUM_DIRS; i++)); do
+    umount "/tmp/meow$i"
+done
+
+# figure out if we have entered the rate limit state
+
+exited_rl=false
+timeout="$(date -ud "2 minutes" +%s)"
+while [[ $(date -u +%s) -le $timeout ]]; do
+    if journalctl -u init.scope | grep "(mount-monitor-dispatch) entered rate limit"; then
+        entered_rl=true
+        break
+    fi
+    sleep 5
+done
+
+# if the infra is slow we might not enter the rate limit state; in that case skip the exit check
+
+if "$entered_rl"; then
```

Comment:
I am not good at shell scripting, but is this condition ok?? This becomes
```
if "true"; then
```
or
```
if "false"; then
```
Does bash correctly handle boolean string??

cc @keszybz.

Reply:
this will work, since /bin/true and /bin/false are programs that just doe "exit(0)" and "exit(1)", but it's a tiny bit too magic for my taste, i'd rather write `if [ "$entered_rl" == 1 ] ; …` here, so that this doesn't mean we fork off a process.

But I am not a shell guru either and don't know what the right coding style here is. @keszybz I think you know this better!

Reply:
`if [ "$entered_rl" = "1" ]` - with one equal sign. Yay bash :-)